package.json

```diff
@@ -35,9 +35,9 @@
         "test": "mocha --require ts-node/register test/unit/**/*.ts",
         "integrationDocumentTest": "node --loader ts-node/esm test/integration/documentTest.tsx",
         "integrationTest": "npx tsc && node test/integration/test.js",
-        "build": "rm -rf dist && npx tsc && mocha --require ts-node/register test/unit/**/*.ts && node test/integration/test.js && eslint . --ext .ts && rollup --config",
         "lint": "eslint . --ext .ts",
-        "allTests": "npx tsc && node test/integration/test.js && mocha --require ts-node/register test/unit/**/*.ts && eslint . --ext .ts"
+        "allTests": "npm run test && npm run integrationTest && npm run integrationDocumentTest && npm run lint",
+        "build": "npm run allTests && rm -rf dist && npx tsc && rollup --config"
```

Comment:
Multiple transpilations. There's npx tsc in allTests (because of integrationTest) and in the build command. Maybe add as an issue separately.

Reply:
oops, will fix.